README.md

```diff
@@ -76,17 +87,16 @@ const handler = {
         - `[host]` - *optional* database host
         - `[dialect]` - *optional* database dialect
         - `[logging = (...msg) => server.log(['trace'], ...mgs)]` - *optional* logging function
-    - `modelsPath` - path to your model definitions
-    - `[models = []]` - *optional* list of models that should be loaded
-        - `[*]` - *(String)* model names; models must be located at `<modelsPath>/<modelName>`
+    - `[models]` - list of objects with the following properties:
+        - `name` - name of the model
+        - `model` - the model definition to load with sequelize
 
 ## model definition
 
 Models should be defined so that they can be imported using [sequelize.import][001]. For convenience
 a `.connection()` function is attached to each model, to access its underlaying sequelize connection.
 
-Also models will be availabe using `server.plugins['hapi-sequelize-models'].models.<modelName>`. Here `<modelName>`
-is not the name defined in `sequelize.define`, but the filename, which is also used in the `databases[*].models[*]`.
+Also models will be available using `server.plugins['hapi-sequelize-models'].models.<modelName>`.
```

Comment:
maybe `...models.<name>`

Reply:
also we could leave a note, that name is the name defined in the config?

Reply:
> that name is the name defined in the config?

what do you mean by that?

Reply:
the comment could look like:

```md
Also models will be availabe using `server.plugins['hapi-sequelize-models'].models.<name>`. Here `<name>` is not the name defined in `sequelize.define`, the name defined in the config.
```